db.json

```diff
@@ -12,5 +12,81 @@
       "content": "Wow such content!",
       "username": "zero_cool"
```

Comment:
Can you also remove the dummy data? Leave only the project mocks; thanks!

Reply:
done